autopush/web/registration.py

```diff
@@ -71,7 +74,19 @@ def convert_chid(self, data):
 class TokenSchema(SubInfoSchema):
     """Filters allowed values from body data"""
     token = fields.Str(allow_none=True)
-    # Temporarily allow 'aps' definition data for iOS.
+
+
+def validate_token(val):
```

Comment:
I was more thinking about someone using a token like " " by accident, and the fact that we don't control tokens, so we don't really have any idea what would be considered a valid size. 

I suppose I could somewhat arbitrarily decide there has to be a minimum of 8 characters and hope someone doesn't send "                ".